### ISSUE TITLE
signup form: text for terms of use checkbox not readable

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/a4-opin#2213]
**URL:** https://opin.me/en/accounts/signup/
**user:** 
**expected behaviour:** 
**behaviour:** The text next to the checkbox is not readable, because it's too light
**important screensize:**
**device & browser:** desktop, firefox
**Comment/Question:** 

Screenshot?
![Screenshot from 2021-06-03 08-50-09](https://user-images.githubusercontent.com/8178179/120600189-c531d980-c448-11eb-90c9-86709d8fb74e.png)

**Comment by fuzzylogic2000** [comment]
fixed with redesign of login page
